rpm/generic/zfs.spec.in

```diff
@@ -92,6 +92,7 @@ Obsoletes:      spl
 # those on either side would conflict with all available documentation.
 Conflicts:      zfs-fuse
 
+BuildRequires:  gcc, make
```

Comment:
This needs to be in the conditional that's the line below for to avoid Debian/Ubuntu breakage.